src/jni_core/src/macros.rs

```diff
@@ -538,8 +538,8 @@ macro_rules! for_all_plain_native_methods {
 /// }
 /// assert_eq!([
 ///     ("f", "int", "(int param1, boolean param2)"),
-///     ("f2", "boolean []", "()"),
-///     ("f3", "java.lang.String", "(byte [] param)")
+///     ("f2", "boolean[]", "()"),
+///     ("f3", "java.lang.String", "(byte[] param)")
```

Comment:
cc @wenym1 for this change

Reply:
looks like a simple fmt in the doc test, a trivial change and we can ignore.